app/code/Dev/ProductComments/Model/Attribute/Backend/Comments.php

```diff
@@ -13,11 +13,11 @@ class Material extends \Magento\Eav\Model\Entity\Attribute\Backend\AbstractBacke
     public function validate($object)
     {
         $value = $object->getData($this->getAttribute()->getAttributeCode());
-        if ( ($object->getAttributeSetId() == 10) && ($value == 'wool')) {
-            throw new \Magento\Framework\Exception\LocalizedException(
-                __('Bottom can not be wool.')
-            );
-        }
+    #            if ($value == 'no') {
```

Comment:
No commented out code. Do you need this backend model at all if it always returns "true"? There is no point having it if it does no validations.